tests/functional/test_cases/dpo-llama3

```diff
@@ -13,10 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+CURRENT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+export SCRIPT_DIR=$CURRENT_DIR/..
```

Comment:
very minor, but can you make the SCRIPT_DIR consistent with the other test_cases?

Reply:
Done. Let me know if you have any other concerns about the way `SCRIPT_DIR` and `DATA_DIR` are set